tslint.json

```diff
@@ -0,0 +1,22 @@
+{
+	"defaultSeverity": "error",
+	"extends": ["tslint:recommended"],
+	"jsRules": {},
+	"rulesDirectory": ["tslint-plugin-prettier"],
```

Comment:
We have to install tslint-plugin-prettier to use that 

https://github.com/prettier/tslint-plugin-prettier